client/src/components/Tool/ToolHelp.vue

```diff
@@ -1,30 +1,93 @@
+<script setup>
+import { computed } from "vue";
+import { getAppRoot } from "onload/loadConfig";
+
+const props = defineProps({
+    content: {
+        type: String,
+        required: true,
+    },
+});
+
+const formattedContent = computed(() => {
+    const node = document.createElement("div");
+    node.innerHTML = props.content;
+
+    const links = node.getElementsByTagName("a");
+    Array.from(links).forEach((link) => {
+        link.target = "_blank";
+    });
+
+    const images = node.getElementsByTagName("img");
+    Array.from(images).forEach((image) => {
+        if (image.src.includes("admin_toolshed")) {
+            image.src = getAppRoot() + image.src;
+        }
+    });
+
+    // loop these levels backwards to avoid increasing heading twice
+    [5, 4, 3, 2, 1].forEach((level) => {
```

Comment:
Would it not be enough to set the css styles for the `.form-help` section and redefine `h1`, `h2` etc locally?

Reply:
No. Simply restyling the headings would hide the issue without solving it. The heading elements in the help texts are breaking the heading level structure and introducing multiple top-level headings to a page. Pushing them down by two levels should fix the structure for most cases and ensures there cannot be multiple h1 elements on the page.

Reply:
@ElectronicBlueberry what is your recommendation for the tool help for tool authors? We should document this and fix our tools if possible. But some recommendations would be nice. Thanks a bunch.

Reply:
@bgruening  My recommendation is that, when using headings, to use h1, h2... headings, without skipping any levels for sub headings. So always start at h1, next sub heading h2, etc.
Essentially treating the headings as if they were in a stand-alone document. (with the exception than multiple h1 tags are fine, because the top-level heading for the help section is provided by the tool from)

I think the best way to document this clearly would be by example "Dos and Don'ts"

Reply:
This is how we handled it in the old hub, too actually -- write each doc to stand alone, and the parent context can shift headings to make it correct when embedded.

Markdown linters (since we're moving away from RST?) can tell you "Hey, looks like your headings are not coherent", if we want more firm guidelines I guess?

Reply:
The tool help is currently in RST ... so I guess we need to adopt the recommendations for RST.
Changing tool help to (optionally) markdown was discussed but never implemented I think, or was it?

Reply:
@bgruening Yeah, my understanding was that we're going to try to move away from RST to MD but I don't know the timeline or even who wants to drive it.